Add getConsentFor (#276)
guardian/consent-management-platform: README.md

});
```

```js
import { getConsentFor } from '@guardian/consent-management-platform';
```

### `getConsentFor(vendor, consent)`

returns: `boolean`

A method to get the consent for each particular custom vendor from Sourcepoint

> params:
> 
> #### `vendor`: string representing the key of the Vendor Id
> Check [VendorIds][] (inside `src/getConsentFor.ts`) to see if the vendor you are trying to use is already configured.
> If not please raise an issue at https://git.io/JUzVL
> 
> [VendorIds]: https://github.com/guardian/consent-management-platform/blob/b243844e2ec2e1c8b0b6849c248c308347de2cd9/src/getConsentFor.ts#L3
> 
> #### `consent`: the consent object been returned from `onConsentChange` callback

Author: ```suggestion
#### `vendor`

type: string

See the list of [`VendorIDs `][] for options. If vendor you need is missing, please [raise an issue](https://git.io/JUzVL) (or a PR!). 

[VendorIDs]: https://github.com/guardian/consent-management-platform/blob/main/src/getConsentFor.ts

#### `consent`

type: `object` 

The consent object returned from `onConsentChange` callback.
```

});
```

```js
import { getConsentFor } from '@guardian/consent-management-platform';
```

### `getConsentFor(vendor, consent)`

returns: `boolean`

A method to get the consent for each particular custom vendor from Sourcepoint

params:

#### `vendor`: string representing the key of the Vendor Id
Check [VendorIds][] (inside `src/types/index.ts`) to see if the vendor you are trying to use is already configured.
If not please raise an issue at https://git.io/JUzVL

[VendorIds]: https://github.com/guardian/consent-management-platform/blob/9de2219cf2499871b3e2f11b4be42acad7ac6f96/src/types/index.ts#L38

#### `consent`: the consent object been returned from `onConsentChange` callback

#### Example

```js

>     import { getConsentFor, onConsentChange } from '@guardian/consent-management-platform`
> 
>     onConsentChange(consent => {
>         const ga = getConsentFor('google-analytics', consent); // true
>         const comscore = getConsentFor('comscore', consent); // false
> 
>         // throws error
>         const eowifnwoeifjoweinf = getConsentFor('eowifnwoeifjoweinf', consent);
> 
>         // you can still use the consent state for a more complicated logic
>         const complexConsentCondition = myComplexConsentTask(consent);
>     })

Author: ```suggestion
import { getConsentFor, onConsentChange } from '@guardian/consent-management-platform`

onConsentChange(consent => {
    const ga = getConsentFor('google-analytics', consent); // true
    const comscore = getConsentFor('comscore', consent); // false

    // throws error
    const eowifnwoeifjoweinf = getConsentFor('eowifnwoeifjoweinf', consent);

    // you can still use the consent state for a more complicated logic
    const complexConsentCondition = myComplexConsentTask(consent);
})
```

});
```

> ## Get consent for particular vendor

Author: maybe this section can be collapsed into `Using Consent`? so it discusses both `getConsentFor` and `onConsentChange`?

```js
import { getConsentFor } from '@guardian/consent-management-platform';
```

### `getConsentFor(vendor, consent)`

returns: `boolean`

A method to get the consent for each particular custom vendor from Sourcepoint

params:

#### `vendor`: string representing the key of the Vendor Id

>     Check VendorIds (inside `src/types/index.ts`) to see if the vendor you are trying to use is already configured.
>     If not please raise an issue at https://git.io/JUzVL

Reviewer: ```suggestion
Check [VendorIds][] (inside `src/types/index.ts`) to see if the vendor you are trying to use is already configured.
If not please raise an issue at https://git.io/JUzVL

[VendorIds]: https://github.com/guardian/consent-management-platform/blob/9de2219cf2499871b3e2f11b4be42acad7ac6f96/src/types/index.ts#L38
```

#### `consent`: the consent object been returned from `onConsentChange` callback

> ### Example

Author: ```suggestion
#### Example
```
